packages/react-components/src/components/Avatar/Avatar.helpers.spec.ts

```diff
@@ -0,0 +1,98 @@
+import {
+  getFontColor,
+  getInitials,
+  getBackgroundColor,
+} from './Avatar.helpers';
+
+describe('getBackgroundColor', () => {
+  it('should generate the same background color for the same initials', () => {
+    const color1 = getBackgroundColor('JD');
+    const color2 = getBackgroundColor('JD');
+    expect(color1).toBe(color2);
+  });
+
+  it('should generate different background colors for different initials', () => {
+    const color1 = getBackgroundColor('JD');
+    const color2 = getBackgroundColor('AB');
+    expect(color1).not.toBe(color2);
+  });
+
+  it('should handle empty initials and return undefined', () => {
+    const color = getBackgroundColor('');
+    expect(color).toBeUndefined();
+  });
+});
+
+describe('getInitials', () => {
+  it('should return initials for a full name', () => {
```

Comment:
It used to be possible to set emoji as customer name, consider checking if such unusual input is handled

Reply:
I think it's not handled in here. I believe there is such an implementation in LiveChat Web

```ts
export function getInitials(name = ''): string {
  const isEmojiInName = EMOJI_REGEX.test(name);
  const finalName = name ? name : '';
  let initials = '';

  initials = finalName
    .split(' ')
    .map((el) => el.charAt(0))
    // eslint-disable-next-line @typescript-eslint/no-unsafe-call
    .filter((el) => isAlphaNumeric(removeDiacritics(el)))
    .join('')
    .substring(0, 2)
    .toUpperCase();

  // fallback for emoji-only and non-alpha-numeric characters
  if (!initials) {
    initials = isEmojiInName
      ? finalName.match(EMOJI_REGEX)?.[0] || ''
      : finalName.toLocaleUpperCase().trim().substring(0, 2);
  }

  return initials;
}
```

@vladko-uxds Do we want to support `emojis` somehow in `Design System`?